queries/perl/injections.scm

```diff
@@ -1,2 +1,17 @@
-((comments) @injection.content
+; an injections.scm file for nvim-treesitter
+((comment) @injection.content
  (#set! injection.language "comment"))
+ 
+((pod) @injection.content
+ (#set! injection.language "pod"))
+
+(
+  (substitution_regexp
+    (replacement) @injection.content
+    (substitution_regexp_modifiers) @_modifiers
+    ; match if there's a single `e` in the modifiers list
+    (#match? @_modifiers "e")
+    (#not-match? @_modifiers "e.*e")
+  )
+  (#set! injection.language "perl")
+)
```

Comment:
keep formatting of parenthesis consistent, on the same line is preferred

Reply:
Apropos: how is the query formatter going? ;)

Reply:
Coming along..just getting really caught up w/ school n' stuff, hope to be done by December :) also it will be upstreamed, and I'm thinking of adding to the cli w/ `tree-sitter query format <path>`